### PR TITLE
Update kekadefaultapp from 1.1.5-rc.1,1.0.0-r.6 to 1.1.22,1.0.1-r.7

### PR DESCRIPTION
**Pull request by hannesbraun**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] The commit message includes the cask’s name and version.

### CODE REVIEW

File: Casks/kekadefaultapp.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 cask 'kekadefaultapp' do
-  version '1.1.5-rc.1,1.0.0-r.6'
-  sha256 'f28224e980afe89af5ee05cf3fa4162fa622554050b0646246597a006960d81d'
+  version '1.1.22,1.0.1-r.7'
```
Comment (vitorgalvao) [inline Casks/kekadefaultapp.rb]: It’s at `1.1.23`.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/kekadefaultapp.rb (hunk 1 above)
Comment (hannesbraun) [reply Casks/kekadefaultapp.rb]: For version `1.1.23`, KekaDefaultApp isn't available and `1.1.22` is also the version linked in their wiki. That's why I updated the cask to `1.1.22`.

Comment (vitorgalvao) [reply Casks/kekadefaultapp.rb]: Ah, got it. I misinterpreted the appcast.

Comment (hannesbraun) [reply Casks/kekadefaultapp.rb]: No problem. Thanks. :)

**Pull request merged**